learntools/machine_learning/nbs/tut1.ipynb

```diff
@@ -0,0 +1,78 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "_cell_guid": "69a3a8be-b3cd-45b5-82bc-333e91e173bf",
+    "_uuid": "fa74e8d20864d22e0c5622cdeb4d695bae301948"
+   },
+   "source": [
+    "**[Machine Learning Course Home Page](kaggle.com/learn/machine-learning).**\n",
+    "\n",
+    "---\n",
+    "# Introduction\n",
+    "We'll start with an overview of how machine learning models work and how they are used. This may feel basic if you've done statistical modeling or machine learning before. Don't worry, we will progress to building powerful models soon.\n",
+    "\n",
+    "The course will have you build models for the following scenario:\n",
+    "\n",
+    "Your cousin has made millions of dollars speculating on real estate. He's offered to become business partners with you because of your interest in data science. He'll supply the money, and you'll supply models that predict how much various houses are worth.\n",
+    "\n",
+    "You ask your cousin how he's predicted real estate values in the past. and he says it is just intuition. But more questioning reveals that he's identified price patterns from houses he has seen in the past, and he uses those patterns to make predictions for new houses he is considering.\n",
+    "\n",
+    "Machine learning works the same way.  We'll start with a model called the Decision Tree. There are fancier models that give more accurate predictions. But decision trees are easy to understand, and they are the basic building block for some of the best models in data science.\n",
+    "\n",
+    "For simplicity, we'll start with the simplest possible decision tree. \n",
+    "\n",
+    "![First Decision Trees](http://i.imgur.com/7tsb5b1.png)\n",
+    "\n",
+    "It divides houses into only two categories. You predict the price of a new house by finding out which category it's in, and the prediction is the historical average price from that category.\n",
+    "\n",
+    "This captures the relationship between house size and price. We use data to decide how to break the houses into two groups,  and then again to determine the predicted price in each group.  This step of capturing patterns from data is called **fitting** or **training** the model. The data used to **fit** the model is called the **training data**.  \n",
+    "\n",
+    "The details of how the model is fit (e.g. how to split up the data) is complex enough that we will save it for later. After the model has been fit, you can apply it to new data to **predict** prices of additional homes.\n",
+    "\n",
+    "---\n",
+    "# Example\n",
```

Comment:
The organization here is a bit confusing. We have a heading ("Example"), then the image, then a hr, then another heading ("Improving the Decision Tree"), which makes it seem like this image and the question that goes along with it are supposed to be some self-contained thing. In actuality it seems like the stuff under the hr is a continuation. I'd suggest putting the "Improving the Decision Tree" heading where "Example" currently is, and removing the hr separator.

Reply:
Indeed.  This (and most issues you identified) were in the live version online too. Going back through everything, I'm amazed how much worse these materials were than I'd thought.